internal/output/driver.go

```diff
@@ -6,9 +6,9 @@ package output
 
 // output drivers
 const (
-	// DriverDefault defines the driver type
-	// when outputting in default format.
-	DriverDefault = "default"
+	// DriverStdout defines the driver type
+	// when outputting in stdout format.
+	DriverStdout = "stdout"
```

Comment:
This is where we update the `default` driver for the `output.Default()` function to `stdout` to match conventions for our `output.Stdout()` function.